### PR TITLE
Sad path's and refactoring, a novel.

### PR DESCRIPTION
**Pull request by Daniel-Gallagher92**
Adds sad path testing,
testing for all uncovered model methods,
cleans up views to match wireframe better,
wraps and assigns div/classes to make CSS hopefully "easier".

All testing is at 100%
Both models and features including sad paths.

If you see any more opportunities for sad path tests, let's get them in there.

x Wrote Tests __x Implemented __x Reviewed

Neccesary checkmarks:

[x] All Tests are Passing

[x] The code will run locally
Type of change

[x] New feature
[] Bug Fix
Implements/Fixes:

description closes #
Check the correct boxes

[x] This broke nothing
[] This broke some stuff
[] This broke everything
Testing Changes

[] No Tests have been changed
[x] Some Tests have been changed and made more robust
[] All of the Tests have been changed(Please describe what in the world happened)
Checklist:

[x] (THERE ARE COMMENTS ASKING OPINIONS IN VIEWS, PLEASE LOOK 👍🏼 )
[x] I have reviewed my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] I have fully tested my code
(REQUIRED)Please Include a link to a gif of your feelings about this branch

Link: 
image

### CODE REVIEW

File: app/views/admin/invoices/show.html.erb [hunk 1]
```diff
@@ -1,27 +1,38 @@
...
-  <p>Created at: <%= @invoice.formatted_date %></p>
+  <p>Created on: <%= @invoice.formatted_date %></p>
   <p>Total Revenue: <%= @invoice.total_revenue_to_currency %>
 
-  <h2>Customer Information:</h2>
-  <p>Customer: <%= @invoice.customer.first_name %> <%= @invoice.customer.last_name %></p>
+  <h2>Customer:</h2>
+  <p><%= @invoice.customer.first_name %> <%= @invoice.customer.last_name %></p>
+
+  <%# Are our customers supposed to have an address? %>
```
Comment (z-fitch) [inline app/views/admin/invoices/show.html.erb]: they shouldn't, its not in the csv

**Review by z-fitch — APPROVED** [review]

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LittleEstyShop7</title>
+    <header>
+      <h1> Little Esty Shop </h1>
+    </header>
```
Comment (ethanrossblack) [inline app/views/layouts/application.html.erb]: In general, the `<head>` tags in an HTML file are for metadata and loading Javascript/CSS files. 

If we want to add "Lil Esty Shop" to every page, that should live inside the body.

File: spec/features/admin/merchants/index_spec.rb [hunk 3]
```diff
@@ -137,11 +150,11 @@
...
-          expect(page).to have_content("Top selling date for #{@merchant_2.name} was #{@merchant_2.best_day.strftime('%B %d, %Y')}")
-          expect(page).to have_content("Top selling date for #{@merchant_4.name} was #{@merchant_4.best_day.strftime('%B %d, %Y')}")
-          expect(page).to have_content("Top selling date for #{@merchant_5.name} was #{@merchant_5.best_day.strftime('%B %d, %Y')}")
+          expect(page).to have_content("Top selling date for #{@merchant_1.name} was #{@merchant_1.best_day.created_at.strftime("%A, %B %-e, %Y")}")
+          expect(page).to have_content("Top selling date for #{@merchant_6.name} was #{@merchant_6.best_day.created_at.strftime("%A, %B %-e, %Y")}")
+          expect(page).to have_content("Top selling date for #{@merchant_2.name} was #{@merchant_2.best_day.created_at.strftime("%A, %B %-e, %Y")}")
+          expect(page).to have_content("Top selling date for #{@merchant_4.name} was #{@merchant_4.best_day.created_at.strftime("%A, %B %-e, %Y")}")
+          expect(page).to have_content("Top selling date for #{@merchant_5.name} was #{@merchant_5.best_day.created_at.strftime("%A, %B %-e, %Y")}")
```
Comment (ethanrossblack) [inline spec/features/admin/merchants/index_spec.rb]: ty for updating all of these dates string methods

File: app/views/admin/invoices/show.html.erb (hunk 1 above)
Comment (ethanrossblack) [reply app/views/admin/invoices/show.html.erb]: let's remove this comment before merging

**Review by ethanrossblack — CHANGES_REQUESTED** [review]
Biggest change is to remove the `<h1>Little Esty Shop</h1>` from the `<head>` of `views/layouts/application.html.erb`

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,7 +1,6 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LittleEstyShop7</title>
```
Comment (ethanrossblack) [inline app/views/layouts/application.html.erb:4]: We still need the line that says "title". That's the website's name that shows up in your browser tab.

**Review by ethanrossblack — CHANGES_REQUESTED** [review]
put title back into the layout

**Review by ethanrossblack — APPROVED** [review]

**Review by JorjaF — APPROVED** [review]
looks good

**Pull request merged**